sdk/vaa/payloads.go

```diff
@@ -249,3 +266,33 @@ func serializeBridgeGovernanceVaa(module string, actionId GovernanceAction, chai
 
 	return buf.Bytes()
 }
+
+func GetIbcConnectionIdBytes(connectionId string) [64]byte {
+	connectionIdBuf := PrependBufferBytesFixed(connectionId, 64)
+	var connectionIdFixedSize [64]byte
+	copy(connectionIdFixedSize[:], connectionIdBuf.Bytes())
+	return connectionIdFixedSize
+}
+
+// Prepends 0x00 bytes to the payload buffer, up to a size of `length`
+func PrependBufferBytesFixed(payload string, length int) *bytes.Buffer {
```

Comment:
`LeftPad` is a better name, I'll update that.

I didn't find a function like this anywhere in our lib. The go-ethereum common package has a `LeftPadBytes` function, but I thought it might be better here to roll our own simple implementation.